core/LSU.cpp

```diff
@@ -750,8 +791,12 @@ namespace olympia
         lsu_flushes_++;
 
         // Flush load/store pipeline entry
-        flushLSPipeline_(criteria);
 
+        // TODO: add logic for single pipeline flush?
```

Comment:
Comment is questionable -- if you have a transaction in flight that needs to be flushed, it technically can be in ANY pipe.